src/observability/input-summary.json

```
@@ -0,0 +1,36 @@
+{"ClusterName":"default","Namespace":"accuknox-agents","ContainerName":"policy-enforcement-agent","ContainerImage":"public.ecr.aws/k9v9d5v2/policy-enforcement-agent:v0.1.0@sha256:005c1fde3ff8a667f3ac7540c5c011c752a7e3aaa2c89aa335703289ed8d80f8","ContainerID":"9af07242b00507a9e7ec1c017a616f6be766f15aa4e6d25133a672a248eb8a5e","PodName":"policy-enforcement-agent-7946b64dfb-lc95w","Operation":"Network","Labels":"app=policy-enforcement-agent","Deployment":"policy-enforcement-agent","Source":"/home/pea/main","Type":"egress","IP":"3.130.218.162","Port":443,"Protocol":"TCP","Action":"Allow","Workload":{"Type":"Deployment","Name":"policy-enforcement-agent"}}
+{"Count":4,"UpdatedTime":1688108538}
```

For the test, we don't need all the summary information. Can we change the input/output format to the following, so that it would be easy to add more test cases.
```suggestion
{
    "test1": [
        {
            "summary": {"PodName":"shared-informer-agent-5cd8b49c55-xhn4z", "Operation":"File", "Source":"/home/sia/main", "Resource":"/var/run/secrets/kubernetes.io/serviceaccount/token"},
            "timeCount": { "Count":4, "UpdatedTime":1688108538}
        },
        {
            "summary": {},
            "timeCount": {}
        },
        {
            "summary": {},
            "timeCount": {}
        },
    ],
    "test2": [],
    "test3": []
}

```

---

Addressed @wazir-ahmed please review